ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/LogTopic.java

```diff
@@ -18,34 +18,35 @@
 
 import org.hyperledger.besu.ethereum.rlp.RLPInput;
 import org.hyperledger.besu.ethereum.rlp.RLPOutput;
-import org.hyperledger.besu.plugin.data.UnformattedData;
-import org.hyperledger.besu.util.bytes.BytesValue;
-import org.hyperledger.besu.util.bytes.DelegatingBytesValue;
 
-public class LogTopic extends DelegatingBytesValue {
+import org.apache.tuweni.bytes.Bytes;
+
+public class LogTopic {
```

Comment:
Once more, but this time in the most relevant spot.  LogTopic should be a Delegating type.  This includes cleanup of all other places LogTopic became a non-specific type.

Reply:
yep fixed.